brake/backends/cachebe.py

```diff
@@ -51,12 +51,23 @@ def _keys(self, func_name, request, ip=True, field=None, period=None):
 
     def count(self, func_name, request, ip=True, field=None, period=60):
         """Increment counters for all relevant cache_keys given a request."""
-        counters = dict((key, 1) for key in self._keys(
```

Comment:
That's a good question. My thinking was that having a default count for all the keys wouldn't be useful, but I need to think about it a little more. I think since all of the test cases manually set the cache, it would avoid the main pitfall I'm worried about with this.